core/src/HIP/Kokkos_HIP_Space.cpp

```diff
@@ -70,6 +71,7 @@ hipStream_t get_deep_copy_stream() {
   }
   return s;
 }
+static std::atomic<int> num_hip_um_allocations(0);
```

Comment:
Should this be `num_hip_managed_allocations`?

Reply:
yeah, you are right, fixed in [0d8d4b](https://github.com/kokkos/kokkos/pull/5112/commits/0d8d4bd904a98998dc9b026a8e145dd144d39e48)